discovery-provider/src/queries/queries.py

```diff
@@ -217,11 +217,11 @@ def get_feed():
     db = get_db()
 
     # filter should be one of ["all", "reposts", "original"]
+    # empty filter value results in "all"
```

Comment:
there might be a more elegant way to do this with a default arg but also might be unnecessary